README.md

```diff
@@ -84,11 +84,11 @@ section of this documentation.
   src="https://github.com/Cognifide/knotx/blob/master/documentation/src/main/wiki/assets/knotx-intro-extensions.png?raw=true"
   alt="Knot.x Extensions"/>
 
-You need to implement custom authentication mechanism for your site and then integrate with service
-talking with its own custom protocol. **Knot.x** is a fully modular platform with very flexible extension
+You need to implement custom authentication mechanism for your site and then integrate with service which 
```

Comment:
_You need to implement **a** custom authentication mechanism for your site and then integrate with **a** service_